### PR TITLE
fix: change all alert messages to custom modal in all screens

### PR DESCRIPTION
**Pull request by JoshM0101**
# Pull Request Review Comments

## Overview
- **Purpose of the PR**:
    - [ ] Change all alert messages to custom modal in all screens

### CODE REVIEW

**Review by edward-stalley-work — CHANGES_REQUESTED** [review]
Thank you Joshua - please see comments.
If they are confusing just let me know and we can have a meeting.

File: client/src/pages/Clients/ClientsListAndEdit.tsx [hunk 1]
```diff
@@ -229,24 +237,27 @@ const ClientsListAndEdit: React.FC = () => {
...
 
     const handleConfirm = async () => {
       // Currently no delete logic
       console.log('Confirmed action for project:', deleteId)
       try {
         const response = await axios.delete(`http://127.0.0.1:8000/api/master-client/${deleteId}/delete/`, {
-          // const response = await axios.get(`http://54.178.202.58:8000/api/master-client/${deleteId}/delete/`, {
+        // const response = await axios.delete(`http://54.178.202.58:8000/api/master-client/${deleteId}/delete/`, {
         })
         setUpdatedClients((prevList) => prevList.filter((client) => client.client_id !== deleteId))
```
[the remark this answers is not in the document]
Comment (JoshM0101) [reply client/src/pages/Clients/ClientsListAndEdit.tsx:256]: Fixed

File: client/src/pages/BusinessDivisions/BusinessDivisionsListAndEdit.tsx [hunk 2]
```diff
@@ -515,6 +528,11 @@ const BusinessDivisionsListAndEdit: React.FC = () => {
         onCancel={closeModal}
         message={translate('deleteMessage', language)}
       />
+      <CrudModal
```
[the remark this answers is not in the document]
Comment (JoshM0101) [reply client/src/pages/BusinessDivisions/BusinessDivisionsListAndEdit.tsx:535]: Fixed

File: client/src/pages/Clients/ClientsListAndEdit.tsx [hunk 3]
```diff
@@ -99,8 +100,8 @@ const ClientsListAndEdit: React.FC = () => {
       })
     }
 
-    const handleSubmit = async (e) => {
-      e.preventDefault()
+    const handleSubmit = async () => {
+      // e.preventDefault()
```
Comment (edward-stalley-work) [inline client/src/pages/Clients/ClientsListAndEdit.tsx:104]: can remove this if not necessary!

File: client/src/pages/BusinessDivisions/BusinessDivisionsListAndEdit.tsx [hunk 4]
```diff
@@ -92,8 +93,8 @@ const BusinessDivisionsListAndEdit: React.FC = () => {
       })
     }
     
-    const handleSubmit = async (e) => {
-      e.preventDefault();
+    const handleSubmit = async () => {
+      // e.preventDefault();
```
Comment (edward-stalley-work) [inline client/src/pages/BusinessDivisions/BusinessDivisionsListAndEdit.tsx:97]: can remove this if not necessary!

File: client/src/pages/Expenses/ExpensesListAndEdit.tsx [hunk 5]
```diff
@@ -109,8 +110,8 @@ const ExpensesList: React.FC = () => {
     setExpensesList(updatedData)
   }
 
-  const handleSubmit = async (e) => {
-    e.preventDefault()
+  const handleSubmit = async () => {
+    // e.preventDefault()
```
Comment (edward-stalley-work) [inline client/src/pages/Expenses/ExpensesListAndEdit.tsx:114]: can remove this if not necessary!

File: client/src/pages/Projects/ProjectsListAndEdit.tsx [hunk 6]
```diff
@@ -165,8 +166,8 @@ const ProjectsListAndEdit: React.FC = () => {
       })
     }
 
-  const handleSubmit = async (e) => {
-    e.preventDefault()
+  const handleSubmit = async () => {
+    // e.preventDefault()
```
Comment (edward-stalley-work) [inline client/src/pages/Projects/ProjectsListAndEdit.tsx:170]: can remove this if not necessary!

**Review by edward-stalley-work — CHANGES_REQUESTED** [review]
Thank you Joshua - please see comments

File: client/src/pages/Clients/ClientsListAndEdit.tsx (hunk 3 above)
Comment (JoshM0101) [reply client/src/pages/Clients/ClientsListAndEdit.tsx]: Removed

File: client/src/pages/BusinessDivisions/BusinessDivisionsListAndEdit.tsx (hunk 4 above)
Comment (JoshM0101) [reply client/src/pages/BusinessDivisions/BusinessDivisionsListAndEdit.tsx]: Removed

File: client/src/pages/Expenses/ExpensesListAndEdit.tsx (hunk 5 above)
Comment (JoshM0101) [reply client/src/pages/Expenses/ExpensesListAndEdit.tsx]: Removed

File: client/src/pages/Projects/ProjectsListAndEdit.tsx (hunk 6 above)
Comment (JoshM0101) [reply client/src/pages/Projects/ProjectsListAndEdit.tsx]: Removed

**Review by edward-stalley-work — APPROVED** [review]

**Pull request merged**